### ISSUE TITLE
Something broken with demo url

### ISSUE THREAD

**Issue by nhonvo** [aviabird/angularspree#417]
Hi team, your demo website is broken, it redirects directly to the casino website.
the url: `https://www.aviacommerce.org/docs/apis-getting-started.html`
![image](https://github.com/user-attachments/assets/3c727f4c-ea74-4fc0-8a8e-79e1048e6e56)

**Comment by phamhaicnttk21** [comment]
yeah i got the same issue. but i dont know the website is attacked or someone else try to redirect to betting domain 
Sir @pkrawat1 @nhonvo

**Comment by pkrawat1** [comment]
Hey 
Thanks for raising the issue. 
I have removed the links for now.
Seems the domain is taken now. Will think about deploying to a diff url.
